source/adapters/hip/program.cpp

```diff
@@ -410,7 +410,12 @@ urProgramGetInfo(ur_program_handle_t hProgram, ur_program_info_t propName,
   case UR_PROGRAM_INFO_KERNEL_NAMES:
     return getKernelNames(hProgram);
   case UR_PROGRAM_INFO_IL:
-    return UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION;
+    // Hip only supports urProgramCreateWithBinary, so we can always return
```

Comment:
I can only see two instances of `Hip` in comments in the adapter. `HIP` is far more common so I propose that.